src/com/github/alexeigrigorev/javatraining/lesson2/TankIAssemblyLIne.java

```diff
@@ -0,0 +1,23 @@
+package com.github.alexeigrigorev.javatraining.lesson2;
+
+public class TankIAssemblyLIne implements IAssemblyLine {
```

Comment:
Переименуй плиз класс TankIAssemblyLIne, убери буквочку I в названии
```suggestion
public class TankAssemblyLIne implements IAssemblyLine {
```